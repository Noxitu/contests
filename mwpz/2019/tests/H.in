3
8 7
4 3
101 1