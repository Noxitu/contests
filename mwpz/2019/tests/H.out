124
22
NO